components/style/gecko/wrapper.rs

```diff
@@ -154,14 +176,15 @@ impl BitOr for GeckoRestyleDamage {
 
 impl<'ln> NodeInfo for GeckoNode<'ln> {
     fn is_element(&self) -> bool {
-        unsafe {
-            Gecko_NodeIsElement(self.0)
-        }
+        use gecko_bindings::structs::nsINode_BooleanFlag;
+        self.0.mBoolFlags & nsINode_BooleanFlag::NodeIsElement as u32 != 0
     }
 
     fn is_text_node(&self) -> bool {
+        // This is a DOM constant that isn't going to change.
+        const TEXT_NODE: u16 = 3;
         unsafe {
-            Gecko_IsTextNode(self.0)
```

Comment:
We'll presumably want a corresponding gecko bug to remove the unused APIs (followup is of course fine).